aws-android-sdk-core/src/main/java/com/amazonaws/ClientConfiguration.java

```diff
@@ -752,40 +734,6 @@ public ClientConfiguration withConnectionTimeout(int connectionTimeout) {
         return this;
     }
 
-    /**
-     * Checks if the {@link IdleConnectionReaper} is to be started
-     *
-     * @return if the {@link IdleConnectionReaper} is to be started
-     */
-    public boolean useReaper() {
```

Comment:
Do we have any sense how many people may currently be using these public methods?

Reply:
It's a little hard to say. I searched around on GitHub. This functionality is also in the V1 Java SDK. I _believe_ that the few bonafide external uses I found were for _that_ SDK.

I can also just `@Deprecated` this method, to be safe. But, I figured it'd be low-impact, even though it's a breaking change. I am planning to bump the release version up to `2.17.x` for the next release, anyway.

Reply:
Yeah sounds good - given that research I think that's fair.